deployment/build-s3-dist.sh

```diff
@@ -273,9 +273,33 @@ echo "--------------------------------------------------------------------------
 # Note: do not install using global (-g) option. This makes build-s3-dist.sh difficult
 # for customers and developers to use, as it globally changes their environment.
 
-STAGE=testEnv
+STAGE=dev
 echo Stage set to $STAGE
 
+# Create sample dev.yaml file with a random user pool domain
+# Remove dev.yaml if it exists, we'll start with a known config that works
+do_cmd rm -f $source_dir/swb-reference/src/config/dev.yaml
```

Comment:
If hardcoding is removed, update this to use $STAGE.yaml

Reply:
cannot remove hardcoded stage since it's needed for CDK synth